planning/behavior_path_planner/include/behavior_path_planner/utils/path_safety_checker/objects_filtering.hpp

```diff
@@ -61,7 +61,8 @@ PredictedObjects filterObjects(
  * @param lim_v Velocity limit for filtering.
  * @return PredictedObjects The filtered objects.
  */
-PredictedObjects filterObjectsByVelocity(const PredictedObjects & objects, double lim_v);
+PredictedObjects filterObjectsByVelocity(
+  const PredictedObjects & objects, const double lim_v, const bool filter_dynamic_objects = true);
```

Comment:
`dynamic_objects` means objects whose velocity is larger than lim_v

Reply:
a little unclear

Reply:
@kosuke55 
I agree.
> a little unclear

How about this?
```
/**
 * @brief Filters objects based on their velocity.
 *
 * Depending on the remove_above_threshold parameter, this function either removes objects with
 * velocities above the given threshold or only keeps those objects. It uses the helper function
 * filterObjectsByVelocity() to do the actual filtering.
 *
 * @param objects The objects to be filtered.
 * @param velocity_threshold The velocity threshold for the filtering.
 * @param remove_above_threshold If true, objects with velocities above the threshold are removed.
 *                               If false, only objects with velocities above the threshold are kept.
 * @return A new collection of objects that have been filtered according to the rules.
 */
PredictedObjects filterObjectsByVelocity(
  const PredictedObjects & objects, const double velocity_threshold,
  const bool remove_above_threshold)
  ```

Reply:
nice! thanks!!